scripts/install/frontend.sh

```diff
@@ -24,6 +24,11 @@ read -r -p "Would you like to install Augur's frontend dependencies? [Y/n] " res
 case "$response" in
   [yY][eE][sS]|[yY]) 
     echo "Installing..."
+
```

Comment:
I sent @ccarterlandis and @gabe-heim a message in slack about this. 

I fixed the augur install-dev and augur install frontend.sh issue … tl;dr, I hacked it a bit because when the backend finishes by calling augur configure generate, it overwrites the logs directory, removing the install folder. My hack for now is to recreate it at the beginning of install/frontend.sh, which allows that to work, and the install to finish.  We are losing the install/backend.log file when augur configure generate is called, which is not ideal, but its good enough for now. There’s enough happening with the GSoC students that getting a release out so we aren’t tripping over each other as much is more important than perfection.  We should fix my hack in a future release.

Reply:
Hmmmm, this didn't even cross my mind, that's definitely what's happening here. Really, the install log files are not very helpful since if the install failed one would want to know right away rather than go check a logfile. They're also immediately discarded the first time Augur is run, so it's not like we even persist them anywhere so they could be useful later. Everything should just be logging to `stdout` but this is fine for now